acslib/ccure/search.py

```diff
@@ -54,9 +56,9 @@ def __init__(
         self.inner_bool = inner_bool.value
         self.term_operator = term_operator.value
         #: List of properties from CCURE to be included in the CCURE response
-        self.display_properties = ["FirstName", "MiddleName", "LastName", "ObjectID"]
+        self.display_properties = []
 
-    def _compile_term(self, term: str) -> str:
+    def _compile_term(self, term) -> str:
```

Comment:
why remove the type hint?

Reply:
I was searching by ObjectID, which is an int, so I didn't want to limit this to just str. And other types like float seem like plausible search terms, so I just left the type open

Reply:
Then we should use a union type hint, to let people know what is allowed and what isn't.

Reply:
Hmm. Okay. thinking more about this you're onto something. No matter what comes in it will be transformed to a string in the lookup. So, nevermind me. This is fine.